README.md

```diff
@@ -3,49 +3,87 @@
 This project is about harvesting from external sources to a CKAN (local or remote) instance.  
 This project uses a custom Python library ([ckan-harvesters](https://pypi.org/project/ckan-harvesters/))in order to read and validate different types of data sources.  
 
-## data.json files
+Install locally
 
-[More info](harvester_ng/datajson/README.md)
+```
+python3 -m venv /path-to-harvester-env
+source /path-to-harvester-env/bin/activate
+pip install -r requirements.txt
+```
 
-## CSW
+## Run with CKAN 
 
-[More info](harvester_ng/csw/README.md)
+First copy `docker-extras/.env.example` to `docker-extras/.env` file.  
+
+Ready to start
+```
+docker-compose up
+```
+
+This starts a group of docker containers including CKAN (and related services) and the Harvester (with Airflow).
+
+In the first run the local CKAN instance will be empty, there is no harvest sources to harvest from.  
+You can add harvest source by impoting them from another CKAN instance (e.g. catalog.data.gov).
+
+Now you can see running:
+ - Airflow: http://localhost:8081/ (without [DAGs](https://airflow.apache.org/docs/stable/concepts.html#dags) by now)
+ - CKAN: http://ckan:5000
```

Comment:
Should be http://localhost:5000

Reply:
> Should be http://localhost:5000

I have troubles to see localhost:5000 from the harvester container but ckan:5000 works fine. Not sure if it's ok to use localhost for all containers. What do you think @akariv?

Reply:
How are these related?
The README comment refers to the user being able to see CKAN running.
It's not about the harvester container accessing the ckan container.

Reply:
Ok, now I understand. Fixed [here](https://github.com/datopian/ckan-ng-harvest/commit/ff868690d1563c966429f1e2dd920217524ba8f1)